kivy_ios/recipes/py3dns/ios.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/DNS/Base.py b/DNS/Base.py
+index 34a6da7..27ab6ca 100644
+--- a/DNS/Base.py
++++ b/DNS/Base.py
+@@ -15,12 +15,13 @@ import socket, string, types, time, select
+ import errno
+ from . import Type,Class,Opcode
+ import asyncore
++import os
+ #
+ # This random generator is used for transaction ids and port selection.  This
+ # is important to prevent spurious results from lost packets, and malicious
+ # cache poisoning.  This doesn't matter if you are behind a caching nameserver
+ # or your app is a primary DNS server only. To install your own generator,
+-# replace DNS.Base.random.  SystemRandom uses /dev/urandom or similar source.  
++# replace DNS.Base.random.  SystemRandom uses /dev/urandom or similar source.
```

Comment:
Why does that line need to be patched?

Reply:
I think PyCharm make some background fixed

Reply:
Ok, consider keeping `patch` files as small as possible, without including any extra un-needed change.
Let me know when the patch is cleaned up.

Reply:
Done